<?xml version="1.0"?>
<launch>  
<param name="use_sim_time" value="false"/>
<arg name="drone_name" default="anafi"/>
<arg name="publish_hz" default="100"/>
<group ns="$(arg drone_name)">
    <node name="sphinx_interface_node" pkg="sphinx_with_gazebo" type="sphinx_interface.py" output="screen" >
        <param name="drone_name" value="$(arg drone_name)" />
        <param name="publish_hz" value="$(arg publish_hz)" />
    </node>
</group>
</launch>
